localstack/runtime/analytics.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import os
+
+from localstack.runtime import hooks
+from localstack.utils.analytics import log
+
+LOG = logging.getLogger(__name__)
+
+TRACKED_ENV_VAR = [
+    "PROVIDER_OVERRIDE_S3",
```

Comment:
```suggestion
```

seems redundant since the loop below filters all envs starting with `PROVIDER_OVERRIDE_` anyway